app/components/memorable_date_component.html.erb

```diff
@@ -30,17 +30,18 @@
                     maxLength: 2,
                     aria: {
                       invalid: false,
-                      describedby: [
-                        "validated-field-error-#{unique_id}",
-                        "validated-field-hint-#{unique_id}",
-                      ],
+                      describedby: "validated-field-error-#{unique_id}",
+                      labelledby: 'month_hint_id',
```

Comment:
A page can potentially have more than one instance of this component, so I think we should use `unique_id` like with the `describedby` to avoid any risk of [duplicate IDs](https://dequeuniversity.com/rules/axe/3.5/duplicate-id).

```suggestion
                      labelledby: "memorable-date-month-hint-#{unique_id}",
```

Reply:
oh wait, i got it working now. I was missing  this syntax: <%={id} %>